Customizing/global/plugins/Services/Repository/RepositoryObject/ReportCompanyGlobal/templates/default/tpl.cat_global_company_report.html

```diff
@@ -1,5 +1,12 @@
-<table class = "fullwidth">
+<div class="ilTableNav yui-skin-sam">
+	<table width="100%" border="0" cellpadding="0" cellspacing="0" datatable="0"><tr>
+	<td style="text-align:left; white-space:nowrap;" >
 
```

Comment:
Bitte kein inline CSS. Das ganze in eine Klasse verpacken und ab in den Skin damit.